### ISSUE TITLE
.npz score file for ucf101

### ISSUE THREAD

**Issue by ZealACMer** [youjiangxu/seqvlad-pytorch#6]
Could you provide us the .npz score file for ucf101, just like the corresponding score files for hmdb51 you have provided in this repository shown as follows:
![image](https://user-images.githubusercontent.com/16794553/59836141-96b2b500-937d-11e9-96ad-adc5db02b46f.png)
What I need is the .npz score file for ucf01(flow), I am grateful for you kind help, thank you very much.

**Comment by youjiangxu** [comment]
Hi,

    I have uploaded our ucf101 score file on the following position:
[ucf_101 score files](https://github.com/youjiangxu/seqvlad-pytorch/tree/master/logits/ucf101).

**Comment by ZealACMer** [comment]
Thank you very much for your prompt reply. The plan of mine is combine my method with the approach you proposed in this paper by late fusion. But I encountered some problem, which is the training list/test list of hmdb51 is different from the lists used in your paper, I wonder if there is a way to do late fusion with the hmdb51 .npz classification score file provided in this repository with the training list/ test list shown in https://github.com/chaoyuaw/pytorch-coviar/tree/master/data/datalists. Thank you so much for your help.

**Comment by youjiangxu** [comment]
Hi,
       I think you could refer to [this page](https://github.com/rohitgirdhar/ActionVLAD#setting-up-the-data), and our renamed list come from [ActionVLAD](https://github.com/rohitgirdhar/ActionVLAD#setting-up-the-data). Some details are as follows:

`NOTE: For HMDB51, I renamed the videos to avoid issues with special characters in the filenames, and hence the numbers in the train/test files. The list of actual filenames is provided in data/hmdb51/train_test_lists/AllVideos.txt, and the new name for each video in that list is the 1-indexed line number of that video. The AllVideos_renamed.txt contains all the HMDB videos that are a part of one or all of the train/test splits (it has fewer entries than AllVideos.txt because some videos are not in any split). So, the video brush_hair/19 in that file (and in the train/test split files) would correspond to the line number 19 in AllVideos.txt.`

**Comment by ZealACMer** [comment]
Thank you so much for your prompt reply. Thanks a lot.
